frigate/ffmpeg_presets.py

```diff
@@ -138,27 +184,54 @@ def parse_preset_hardware_acceleration_decode(arg: Any) -> list[str]:
     return PRESETS_HW_ACCEL_DECODE.get(arg, None)
 
 
+def _parse_rotation_scale(
+    arg: Any,
+    mode: str,
+    rotate: int,
+) -> str:
+    """Return the correct rotation scale or "" if preset none is set."""
+    if not isinstance(arg, str) or " " in arg:
+        return ""
+
+    if rotate == 90:
+        transpose = "clock"
+    elif rotate == 180:
+        if arg.startswith("preset-vaapi") or arg.startswith("preset-intel-qsv"):
+            transpose = "reverse"
+        else: # No 'reverse' option suported, then 2 'clocks' rotations
+            transpose =  "clock,transpose=clock"
+    elif rotate == 270:
+        transpose =  "cclock"
+    else : # Rotation not need or not supported
+        return ""
+
+    return PRESETS_HW_ACCEL_SCALE_ROTATION.get(arg, "").get(mode).format(transpose)
```

Comment:
How does this work with the second .get call? This would fail if calling .get on an empty string (which is the default value for no preset)